app/controllers/api/v2/admin_controller.rb

```diff
@@ -44,11 +44,16 @@ def search_users
     @username = data.fetch('username', '')
     @email = data.fetch('email', '')
 
-    @users, @paginate = paginate User.where(
-      'UPPER(username) = UPPER(?) or UPPER(email) = UPPER(?)',
-      @username.to_s,
-      @email.to_s
-    )
+    @query = User.all
+    if @username.present?
```

Comment:
Should we remove the `fetch`, I don't know it's purpose?

```suggestion
    if data['username'].present?
```

Reply:
I think in this case it's purpose is to provide a fallback (an empty string '') if the 'username' is not found.
We don't need it since the `.present?` method covers it!